modules/standard/Random.chpl

```diff
@@ -896,6 +899,8 @@ module Random {
 
         :arg n: The position in the stream to skip to.  Must be > 0.
         :type n: `integral`
+
+        :throws IllegalArgumentError: When called with non-positive 'n' value.
```

Comment:
Since this will impact how the docs are rendered, I'm going to be a stickler and ask you to match the formatting strategy as closely as possible :) Which means using back tic (`) instead of single quotation marks (') around n.